valkyrie/lib/valkyrie/persistence/solr/queries/find_members_query.rb

```diff
@@ -25,15 +26,18 @@ def unordered_members
     end
 
     def docs
-      connection.get("select", params: { q: query, rows: 1_000_000_000 })["response"]["docs"]
+      options = { q: query, rows: 1_000_000_000 }
+      options[:fq] = "{!raw f=internal_resource_ssim}SecondResource" if model
+      connection.get("select", params: options)["response"]["docs"]
     end
 
     def member_ids
       Array.wrap(resource.member_ids)
     end
 
     def query
-      "{!join from=member_ids_ssim to=id}id:#{id}"
+      q = "{!join from=#{MEMBER_IDS} to=id}id:#{id}"
```

Comment:
Why the assignment to `q`?

Reply:
Uh, that should have been removed, I was mutating it an earlier iteration.